tests/integration/golang/mlflow/experiment/search_test.go

```diff
@@ -1,3 +1,262 @@
 //go:build integration
 
 package experiment
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/hetiansu5/urlquery"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/api"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/api/request"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/api/response"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/dao/models"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/fixtures"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type SearchExperimentsTestSuite struct {
+	suite.Suite
+	client   *helpers.HttpClient
+	fixtures *fixtures.ExperimentFixtures
+}
+
+func TestSearchExperimentsTestSuite(t *testing.T) {
+	suite.Run(t, new(SearchExperimentsTestSuite))
+}
+
+func (s *SearchExperimentsTestSuite) SetupTest() {
+	s.client = helpers.NewHttpClient(os.Getenv("SERVICE_BASE_URL"))
+	fixtures, err := fixtures.NewExperimentFixtures(os.Getenv("DATABASE_DSN"))
+	assert.Nil(s.T(), err)
+	s.fixtures = fixtures
+}
+
+func (s *SearchExperimentsTestSuite) Test_Ok() {
+	// 1. prepare database with test data.
+	experiments := []models.Experiment{
+		{
+			Name: "Test Experiment 1",
```

Comment:
please do not use `whitespaces` in test names. When test will fail, all `whitespaces` will be changed to underscores, so when you will try to find this test by test name and copy past test name with underscores you wont find it.